cloud_experts_tutorials/cloud-experts-dynamic-certificate-custom-domain.adoc

```diff
@@ -42,22 +47,28 @@ $ export REGION=$(oc get infrastructure cluster -o=jsonpath="{.status.platformSt
 $ export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
 $ export SCRATCH="/tmp/${CLUSTER_NAME}/dynamic-certs"
 $ mkdir -p ${SCRATCH}
-$ echo "Cluster: ${CLUSTER_NAME}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
 ----
 <1> The custom domain.
 <2> The e-mail Let's Encrypt will use to send notifications about your certificates.
+. Ensure all fields output correctly before moving to the next section:
++
+[source,terminal]
+----
+$ echo "Cluster: ${CLUSTER_NAME}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
+----
 
+[id="cloud-experts-dynamic-certificate-prep-aws"]
 == Preparing your AWS account
 
-When cert-manager requests a certificate from Let’s Encrypt (or another ACME certificate issuer), Let's Encrypt servers validate that you control the domain name in that certificate using “challenges". For this tutorial, you are using a link:https://letsencrypt.org/docs/challenge-types/#dns-01-challenge[DNS-01 challenge] that proves that you control the DNS for your domain name by putting a specific value in a TXT record under that domain name. This is all done automatically by cert-manager. To allow cert-manager permission to modify the Amazon Route 53 public hosted zone for your domain, you need to create an IAM role with specific policy permissions and a trust relationship to allow access to the pod.
+When cert-manager requests a certificate from Let’s Encrypt (or another ACME certificate issuer), Let's Encrypt servers validate that you control the domain name in that certificate using _challenges_. For this tutorial, you are using a link:https://letsencrypt.org/docs/challenge-types/#dns-01-challenge[DNS-01 challenge] that proves that you control the DNS for your domain name by putting a specific value in a TXT record under that domain name. This is all done automatically by cert-manager. To allow cert-manager permission to modify the Amazon Route 53 public hosted zone for your domain, you need to create an Identity Access Management (IAM) role with specific policy permissions and a trust relationship to allow access to the pod.
 
 The public hosted zone that is used in this tutorial is in the same account as the ROSA cluster. If your public hosted zone is in a different account, a few additional steps for link:https://cert-manager.io/docs/configuration/acme/dns01/route53/#cross-account-access[Cross Account Access] are required.
```

Comment:
The same AWS account? The same Red Hat account? Something else?

Reply:
@michaelryanmcneill I think this refers to a Red Hat account- can you confirm please?

Reply:
Same AWS account.